app/storage/storageConfig.js

```diff
@@ -36,8 +41,15 @@ class StorageConfig {
         connection.locationConstraint = env['S3_' + metro + '_LOCATION_CONSTRAINT'];
         connection.s3ForcePathStyle = true;
         connection.signatureVersion = 'v4';
-        connection.channelBucket = env['S3_' + metro + '_CHANNEL_BUCKET'] || env.S3_CHANNEL_BUCKET || 'razee';
         connection.resourceBucket = env['S3_' + metro + '_RESOURCE_BUCKET'] || env.S3_RESOURCE_BUCKET || connection.channelBucket || 'razee';
+        connection.orgBucketPrefix = env['S3_' + metro + '_ORG_BUCKET_PREFIX'] || env.S3_ORG_BUCKET_PREFIX || 'razee-org-';
+        let kmsEnabled = (env['S3_' + metro + '_KMS_ENABLED'] || env.S3_KMS_ENABLED || null);
+        kmsEnabled = !!(kmsEnabled && !_.includes(['false', '0'], kmsEnabled));
```

Comment:
app/conf.js is using `process.env.KMS_COS_ENABLED`, here it's checking both `S3_[metro]_KMS_ENABLED` and `S3_KMS_ENABLED` -- there should be only one env var for whether KMS is enabled.

app/conf.js is also using slightly simpler code to check if the ENABLED value is true-ey or false-ey, that doesn't require using `let` or lodash.  Should be the same in both places (and simpler version preferred personally, so I don't have to do so much mental counting of `!`s to understand what's going on :-D )